x/authority/keeper/authorization_list_test.go

```diff
@@ -47,3 +49,184 @@ func TestKeeper_SetAuthorizationList(t *testing.T) {
 		require.Equal(t, newAuthorizationList, list)
 	})
 }
+
+func TestKeeper_CheckAuthorization(t *testing.T) {
```

Comment:
The tests look good but I would suggests adding some more elements in some of the authorizationList and policies to add more entropy
Or just an additional case with successful check but many component in the list